### DISCUSSION TITLE
Installing dependencies build errors ???

### DISCUSSION THREAD

**Discussion by gslender** [codemagic-ci-cd/codemagic-docs#1668] [Q&A] [answered]
Hi anyone/everyone,

I'm using Codemagic for a Flutter build that works / builds fine for Android and iOS, but I've noticed that in the Installing dependencies section it claims the build failed, but it continues regardless. Without really understanding what's happening, it looks like this is related to Android dependancy / build caching, but as the script/code is owned by Codemagic team, I cannot be 100% sure.

Any thoughts on what this below is saying and why the support team at Codemagic are unable to help/figure it out??? 

I've asked many times but they keep suggesting its a build in my code/app, but it isn't - I can't build this on my setup fine - plus it builds on Codemagic fine it just has this error in the logs!?

```
> /Users/builder/programs/gradle-7.3.1/bin/gradle wrapper --gradle-version 7.5 --distribution-type all
Starting a Gradle Daemon (subsequent builds will be faster)
FAILURE: Build failed with an exception.
* Where:
Build file '/Users/builder/clone/android/app/build.gradle' line: 24
* What went wrong:
A problem occurred evaluating project ':app'.
> Failed to apply plugin 'com.android.internal.version-check'.
> Minimum supported Gradle version is 7.4. Current version is 7.3.1. If using the gradle wrapper, try 
 editing the distributionUrl in /Users/builder/clone/android/gradle/wrapper/gradle-wrapper.properties
 to gradle-7.4-all.zip
* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
* Get more help at https://help.gradle.org/
BUILD FAILED in 15s
```

**Comment by gslender** [comment]
@himesh-cm is this not the same as #1193 ?

**Reply by himesh-cm** [reply]
Hi @gslender , Nope, it's not.
We are in touch with devs regarding this gradle check and will keep you posted.

**Reply by rafeysup93** [reply, marked as answer]
Hi @gslender, newer versions of Gradle have been installed on Codemagic.
